Duty List Helper/Helper 1/readme.md

```diff
@@ -0,0 +1,2 @@
+<h2>Helper1:</h2> 
```

Comment:
Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8389313)
```suggestion
<h2>Helper1:</h2>
```